### PR TITLE
Add per-commit runs & PR trigger phrases for Pri1 R2R GCStress 15 test runs for Windows, OSX, CentOS, and Ubuntu

### PR DESCRIPTION
**Pull request by wtgodbe**
Trigger phrases:

x64:

"test {os} gcstress15_r2r {scenario} please."

os can be {Ubuntu, OSX, CentOS7.1, Windows_NT}
scenario can be {Release, Checked}

x86:

"test Windows_NT x86 Release gcstress15_r2r please."

### CODE REVIEW

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Comment by wtgodbe** [comment]
@gkhanna79 PTAL

File: netci.groovy [hunk 1]
```diff
@@ -66,7 +66,7 @@ class Constants {
                'gcstress0xc_minopts_heapverify1' : ['COMPlus_GCStress'  : '0xC', 'COMPlus_JITMinOpts'  : '1', 'COMPlus_HeapVerify'  : '1']
                ]
     // This is the basic set of scenarios
-    def static basicScenarios = ['default', 'pri1', 'ilrt', 'r2r', 'pri1r2r']
+    def static basicScenarios = ['default', 'pri1', 'ilrt', 'r2r', 'pri1r2r', 'gcstress15_r2r']
```
Comment (gkhanna79) [inline netci.groovy]: Can you please make it "gcstress15_pri1r2r"?

**Comment by gkhanna79** [comment]
Modulo a comment, LGTM

**Comment by wtgodbe** [comment]
@gkhanna79 sure thing, the switch has been made.

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please.

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please.

**Comment by gkhanna79** [comment]
LGTM

@RussKeldorph - FYI

**Comment by wtgodbe** [comment]
Generator PR test passed (hasn't turned green yet on the PR, but it did pass - http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/GenPRTest/job/generator_prtest/28/), so I'm merging.

**Comment by gkhanna79** [comment]
Thank you. Can you please schedule a dummy PR to issue Pri1R2R GCStress?

**Pull request merged**